web/app/js/components/TopRoutes.jsx

```diff
@@ -68,19 +68,19 @@ class TopRoutes extends React.Component {
       namespace: '',
       to_name: '',
       to_type: '',
-      to_namespace: ''
+      to_namespace: '',
     }, _pick(props.query, Object.keys(topRoutesQueryProps)));
 
 
     this.state = {
-      query: query,
+      query,
```

Comment:
How come there's no `key: value` here?

Reply:
This was caught by the [object-shorthand](https://eslint.org/docs/rules/object-shorthand) rule. ES6 allows a shorthand that uses the variable name as the property name if it's omitted, and this rules makes sure that's used consistently throughout.